src/main/java/com/mongodb/jdbc/MongoDatabaseMetaData.java

```diff
@@ -864,76 +710,23 @@ public ResultSet getProcedureColumns(
         metaDoc.columns.add(new Column("", "", "", "SPECIFIC_NAME", "SPECIFIC_NAME", "string"));
 
         docs.add(metaDoc);
-        return new MongoResultSet(null, new MongoExplicitCursor(docs), true);
+        return new MySQLResultSet(null, new MongoExplicitCursor(docs), true);
     }
 
-    private String patternCond(String colName, String pattern) {
+    protected String patternCond(String colName, String pattern) {
```

Comment:
Since this is specific to MySQL, I'd just move it into the subclass.

Reply:
Good catch. I went a little bulk-edit-happy and didn't fully inspect all of the helper methods I changed from `private` -> `protected`.

Reply:
done